unit_tests/falco/test_configuration.cpp

```diff
@@ -37,6 +37,9 @@ static std::string sample_yaml =
 	"    - elem2\n"
 	"    - elem3\n";
 
+static std::vector<std::string> loaded_conf_files;
+static std::vector<std::string> loaded_conf_warnings;
```

Comment:
cool, now that we have a warnings vector we can assert that these tests correctly throw the right warning where needed, WDYT?

Reply:
I'd just assert the expected length (not the actual strings), so that they are more resilient to string changes.